bin/yaml/libraries.yaml

```diff
@@ -1658,6 +1670,19 @@ libraries:
       targets:
       - 1.8.0
       type: github
+    hip-runtime-amd:
+      check_file: include/hip/hip_runtime.h
+      path_name: libs/rocm/{name}
+      s3_path_prefix: hip-amd-rocm-{name}
+      subdir: libs/rocm/{name}
+      untar_dir: opt/compiler-explorer/libs/rocm/{{name}}
+      targets:
+      - 4.5.2
+      - 5.0.2
+      - 5.1.3
+      - 5.2.3
+      install_always: true
```

Comment:
@mattgodbolt or @partouf can you remind me what the side-effects of "install_always" are again? 

Hopefully this doesn't mean it automatically links against this library by default as that would be undesirable.

Reply:
install_always means it's always installed even if we already have it. That's generally a bad idea unless it's a trunk version (but all trunk/nightly things are automatically tagged as install_always)
So it really shouldn't be there.

Reply:
That's different from what I thought it did. Thanks for the correction!